docs/root/start/quick-start/run-envoy.rst

```diff
@@ -82,6 +82,10 @@ Run Envoy with the demo configuration
 
 The ``-c`` or ``--config-path`` flag tells Envoy the path to its initial configuration.
 
+.. note::
+
+   The configuration file extension must be ``yaml``.
```

Comment:
im not sure if it does - i think perhaps its parsed based on the file ending - ie yaml or yml should work - and im guessing the default is json

Reply:
docs are here https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-c 

i agree this could be clearer but it needs to also be accurate

Reply:
I test with yml, it fails.

How can I make it accurate?